q2_fmt/_peds.py

```diff
@@ -253,15 +274,15 @@ def _compute_peds(peds_df: pd.Series, peds_type: str, peds_time: int,
             'title': subject_column,
             'description': 'Subject IDs linking samples across time'
         })
-        peds_df["transfered_donor_features"].attrs.update({
-            'title': "Transfered Donor Features",
+        peds_df[transfered].attrs.update({
+            'title': "Transfered Reference Features",
             'description': '...'
         })
-        peds_df['total_donor_features'].attrs.update({
-            'title': "Total Donor Features",
+        peds_df[total].attrs.update({
+            'title': "Total Reference features",
```

Comment:
```suggestion
            'title': "Total Reference Features",
```
Should this be capitalized as well?